app.js

```diff
@@ -0,0 +1,127 @@
+const statusDisplay = document.querySelector('.game--status');
+const board = document.querySelector('.board');
+const form = document.getElementById('form');
+const alert = document.querySelector('.alert');
+let gameActive = true;
+
+function Player(name, symbol) {
+  this.name = name;
+  this.symbol = symbol;
+}
+
+const playerOne = new Player('', 'X');
```

Comment:
- [ ] It doesn't use global code - encapsulate logic inside modules or factories

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png)